hicommonwealth/commonwealth
# Remove is_default from Profiles

## Link to Issue
Closes: #5229 

## Description of Changes
- Removes `is_default` column from Profiles table, and references in code. This did not serve any real purpose, since we never implemented multiple Profiles per user.
- Updates db-doc schema

## "How We Fixed It"


## Test Plan
- CA (click around) tested on local and frack:
  - Sign in from the homepage
    - Expect your normal sign-in experience (Profile details and linked addresses are the same)
  - sign out
  - Go to a community page like /osmosis or /dydx and sign in. Go to "Edit Profile" from dropdown.
    - Expect your normal sign-in experience (Profile details and linked addresses are the same)
    - If this a new community for you, expect normal New Profile flow.
   

## Deployment Plan

1. 

## Other Considerations

-
